pkgs/applications/video/deface/default.nix

```
@@ -39,16 +41,20 @@ python3.pkgs.buildPythonApplication rec {
   ];
 
   patchPhase = ''
```

```suggestion
  postPatch = ''
```
We should not override the patch phase.

---

Indeed